src/frontends/paddle/src/op/dequantize_linear.cpp

```diff
@@ -0,0 +1,78 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "default_opset.hpp"
+#include "openvino/frontend/paddle/node_context.hpp"
+
+namespace ov {
+namespace frontend {
+namespace paddle {
+namespace op {
+NamedOutputs dequantize_linear(const NodeContext& node) {
+    // extract the INPUTS
+    const auto x = node.get_input("X");
+    const auto scale = node.get_input("Scale");  // type: float or 1-D
+    const auto zero_point = node.get_input("ZeroPoint");
```

Comment:
> `zero_point` is not used in graph construction. I believe it should be used to `Subtract` from converted `x`.

The PaddleSlim just supports the Symmetry quantization so the zero_point is always equal to 0.

Reply:
But if one day they start to support non zero `zero_point`, how are you going to detect it? The code doesn't have any checks for that. BTW, non zero `zero_point` doesn't always mean asymmetric quantization, it may be because they are using u8 and setting some non-zero zero point even in case of truly symmetric quantization.

Reply:
> 

Double check it. PaddleSlim just maps the value to [-128,127] and keeps the zero_point equal to zero. So it is just designed on purpose here.

Reply:
@slyalin 
Hi, in order to guarantees the completeness of the function, I design it to support the zero_point.
As talking to PDPD team, they don't support U8 and have no flag to mark it.
So, I have to implement using I8 formula here. Also see it in the pass.